pitaya-sharp/NPitaya/src/PitayaCluster.API.cs

```diff
@@ -94,6 +94,10 @@ static void HandleRpcCallback(IntPtr userData, IntPtr ctx, IntPtr rpc)
             {
                 Logger.Error("Failed to decode request, error:{0}", e.Message);
             }
+            finally
+            {
+                pitaya_ctx_drop(ctx);
```

Comment:
You probably need to call `pitaya_rpc_drop` as well, no?

Reply:
I looked into it with him and rpc is already dropped inside pitaya_rpc_respond that is called in DispatchRpc function before

Reply:
Exactly.